support-frontend/assets/pages/digital-subscription-landing/components/paymentSelection/paymentSelection.jsx

```diff
@@ -1,13 +1,13 @@
 // @flow
 
 import React from 'react';
-import { connect } from 'react-redux';
 // components
 import ProductOption, {
   type Product,
 } from 'components/product/productOption';
 import {
-  mapStateToProps,
+  getPaymentOptions,
```

Comment:
I definitely prefer that the exports from modules such as `helpers/paymentSelection` are now named in a way which indicates their purpose, rather than a redux api method name, much more readable!